rfcs/text/0000-roadmaps+planning.md

```diff
@@ -0,0 +1,435 @@
+* Documenting important decisions, discussions, and questions 
+* Assigning action items to move toward the creation of the Roadmap, should no Project Lead volunteer, and following up with them through completion
+
+### Drafting the Roadmap
+[Drafting the Roadmap]:#drafting-the-roadmap
+
+Project Leads MUST independently create Themes by utilizing the input provided in the Science Roadmap, Technical Objectives RFC, DCP User Survey and UX Insights, and Roadmap Retrospective.
+
+To create the draft Product Roadmap, the Roadmap Shepherd:
+- Forks the *HumanCellAtlas dcp-community* repository
+- Copies `rfcs/roadmap-template.md` to `rfcs/text/0000-roadmap-YYYY.md` - where "YYYY" is the target year for the roadmap. 
+
+The Roadmap Shepherd MUST collate all themes into the draft which is an Informational RFC. 
+
+Project Leads MUST review the Themes as a collective and distill the revised list to a lesser number of themes. This is achieved through a consensus building process, which is implemented by the Roadmap Shepherd and occurs during specifically noted Project Lead meetings, outlined below:
+
+* Explore the Themes, search for commonalities, and document concerns and issues 
+* Identify potential consolidated or alternative Themes, resolution for concerns and issues, and action items required for further understanding
+* Prioritize the Themes within the Roadmap
+* Update the Roadmap to reflect revised content
+* Resolve outstanding questions and concerns
+
+If the Project Leads fail to reach consensus within the specified timebox, then the draft MUST be escalated to Oversight for consultation.
+
+When rough consensus is reached, the Roadmap is submitted as a pull request to the *dcp-community* repo and follows the DCP RFC process for a *Governance* RFC starting with [community review](https://github.com/HumanCellAtlas/dcp-community/blob/master/rfcs/text/0001-rfc-process.md#reviewing-rfcs). 
+
+The Roadmap Shepherd MUST be assigned as the Shepherd for the RFC and ensure that the Project Leads (Authors) respond to comments, answer questions, and update the Roadmap RFC as required.
+
+### Iterating with Oversight
+[Iterating with Oversight]: #iterating-with-oversight
+
+Roadmaps MUST be approved by Oversight rather than DCP PM which is a minor variation from the current RFC Approval process for Governance RFC(s).
+
+The Roadmap Shepherd MUST send the draft Roadmap to Oversight a week in advance of a review meeting between the Project Leads and Oversight.
+
+The Project Leads MUST iterate with Oversight to review and revise the draft Roadmap until it meets with Oversight approval.
+
+Once approved, the Roadmap Shepherd will follow the normal RFC process for [approved RFC(s)](https://github.com/HumanCellAtlas/dcp-community/blob/master/rfcs/text/0001-rfc-process.md#approving-rfcs).
+
+### Updating the Roadmap
+
+The Roadmap MAY be updated with Project Lead consensus and Oversight approval for a number of reasons including but not limited to:
```

Comment:
If we're doing a yearly roadmap then we should plan on updating on a quarterly basis.

Reply:
Agreed. I will modify this section to reflect that suggestion.